beaker-tests/Sanity/copr-cli-basic-operations/all-in-tmux-redhat.sh

```diff
@@ -3,7 +3,8 @@
 our_dir=$(readlink -f "$(dirname "$0")")
 
 OWNER=$(copr-cli whoami)
-FRONTEND_URL=https://dev-copr.devel.redhat.com
+FRONTEND_HOST=dev-copr.devel.redhat.com
```

Comment:
You sure we want this URL in the upstream code?

Reply:
We don't, but I was lazy to change this for now